KubeArmor/eventAuditor/entryPointHandler.go

```diff
@@ -16,48 +17,83 @@ import (
 // InitializeEntryPoints Function
 func (ea *EventAuditor) InitializeEntryPoints() bool {
 	// if something wrong, return false
+	b, err := bpf.OpenObjectFromFile("entrypoint.bpf.o")
```

Comment:
path? KubeArmor/eventAuditor? KubeArmor/BPF?

Reply:
If the object file is generated in the same folder (KubeArmor/eventAuditor), then we don't need to specify the whole path(in the above scenario) . But if we want it to be in KubeArmor/BPF, then we will need to specify the whole path. Should I generate the object file in KubeArmor/BPF?

Reply:
Please refer to the https://github.com/kubearmor/KubeArmor/tree/event-auditor/KubeArmor/BPF as the current place for C bpf files.

And for working with them please refer to KABPFManager https://github.com/kubearmor/KubeArmor/blob/97d4af10746499837b4614893ccb6c186d984a80/KubeArmor/eventAuditor/eventAuditor.go#L20

Reply:
> Please refer to the https://github.com/kubearmor/KubeArmor/tree/event-auditor/KubeArmor/BPF as the current place for C bpf files.
> 
> And for working with them please refer to KABPFManager
> 
> https://github.com/kubearmor/KubeArmor/blob/97d4af10746499837b4614893ccb6c186d984a80/KubeArmor/eventAuditor/eventAuditor.go#L20

Okay, I will update the same. Thanks

Reply:
good! we need to maintain all C files in KubeArmor/BPF.

By the way, guys, we need to be careful in terms of the BPF file location and its compilation
since we cannot compile the C files in the KubeArmor image,
meaning that the C files need to be compiled in a builder container (during the KubeArmor image creation)
and we will only use the object files in the final KubeArmor image.